docs/skill.md

```diff
@@ -16,6 +16,8 @@ This means it is possible to, at any point, grab the `context` and have access t
 
 For example, in the `ErrorHandler(Handler)` class, the code often grabs a reference to its context and by doing so can access initialised and running framework objects such as an `OutBox` for putting messages into.
 
+Moreover, you can read/write to the _agent context namespace_ by accessing the attribute `SkillContext.namespace`.
+
```

Comment:
Too short? :-)

Reply:
Ok, maybe should also explain how one feeds objects to it via the builder?